### ISSUE TITLE
jesse 1.5.4 hex package published incorrectly?

### ISSUE THREAD

**Issue by nalundgaard** [for-GET/jesse#88]
Hi, I am looking at https://hex.pm/packages/jesse/1.5.4 and it says there are 0 dependencies:

<img width="511" alt="Screen Shot 2020-04-06 at 16 23 39" src="https://user-images.githubusercontent.com/1851445/78606470-fef65880-7822-11ea-90f6-24c677b9c63e.png">

I'm not sure what flaw caused this to be published as such, but it should display both `jsx` and `rfc3339` (example: https://hex.pm/packages/erlcloud/3.2.18). Is there a chance that it was published with an outdated version of rebar3?

**Comment by seriyps** [comment]
Oh, that's weird..
Looking at https://diff.hex.pm/diff/jesse/1.4.0..1.5.4 I can see that they are mentioned in rebar.lock , but not in hex_metadata.config.

I use rebar3 3.13.0 and it's pretty much fresh version.

Does this cause any problems?

**Comment by seriyps** [comment]
Tried to run hex publish again:

```
$ rebar3 hex publish
===> Verifying dependencies...
Publishing jesse 1.5.4 to hexpm
  Description: jesse (JSon Schema Erlang) is an implementation of a JSON Schema validator for Erlang.
  Dependencies:
    
  Included files:
    LICENSE
    NOTICE
    README.md
    rebar.config
    rebar.lock
    src/jesse.app.src
    src/jesse.erl
    src/jesse_cli.erl
    src/jesse_database.erl
    src/jesse_error.erl
    src/jesse_json_path.erl
    src/jesse_lib.erl
    src/jesse_schema_validator.erl
    src/jesse_schema_validator.hrl
    src/jesse_state.erl
    src/jesse_tests_util.erl
    src/jesse_validator_draft3.erl
    src/jesse_validator_draft4.erl
  Licenses: Apache 2.0
  Links:
    Github: https://github.com/for-GET/jesse
  Build tools: rebar3
Be aware, you are publishing to the public Hexpm repository.
Before publishing, please read Hex CoC: https://hex.pm/policies/codeofconduct
Proceed? ("Y")> 
```

it shows "dependencies" as empty.... That's weird... Can you try to run `rebar3 hex publish`? What does it show?

**Comment by seriyps** [comment]
So far it looks like rebar3_hex bug. I reported it upstream.

**Comment by seriyps** [comment]
Ok, I published 1.5.5. It looks correct.

**Comment by nalundgaard** [comment]
Awesome, I concur. Thanks!
